src/iuse.cpp

```diff
@@ -2131,6 +2131,52 @@ int iuse::unpack_item( player *p, item *it, bool, const tripoint & )
     return 0;
 }
 
+int iuse::pack_cbm( player *p, item *it, bool, const tripoint & )
+{
+    item_location bionic = g->inv_map_splice( []( const item & e ) {
+        return e.is_bionic();
+    }, _( "Choose CBM to pack" ), PICKUP_RANGE, _( "You don't have any CBMs." ) );
+
+    if( !bionic ) {
+        return 0;
+    }
+    if( !bionic.get_item()->faults.empty() ) {
+        if( p->query_yn( _( "This CBM is faulty.  You should mend it first.  Do you want to try?" ) ) ) {
+            p->mend_item( std::move( bionic ) );
+        }
+        return 0;
+    }
+
+
+    if( !bionic.get_item()->has_flag( "NO_PACKED" ) ||
+        bionic.get_item()->has_flag( "PACKED_FAULTY" ) ) {
+        if( !p->query_yn( _( "This CBM is already prepared.  Do you want to re-do it?" ) ) ) {
+            return 0;
+        }
+    }
+
+    const int success = p->get_skill_level( skill_firstaid ) - rng( 0, 6 );
+    if( success > 0 ) {
+        bionic.get_item()->unset_flag( "NO_PACKED" );
+        bionic.get_item()->unset_flag( "PACKED_FAULTY" );
+        add_msg( m_info, _( "You carefully prepare the CBM for sterilization." ) );
+    } else {
+        bionic.get_item()->unset_flag( "NO_PACKED" );
+        bionic.get_item()->set_flag( "PACKED_FAULTY" );
+        if( success == 0 ) {
+            add_msg( m_bad, _( "You're not sure about the quality of your work." ) );
+        } else {
+            add_msg( m_info, _( "You put the CBM in the pouch and close it." ) );
+        }
```

Comment:
```
add_msg( m_info, _( "You put the CBM in the pouch and close it." ) );
if( success == 0 ) {
    add_msg( m_bad, _( "You're not sure about the quality of your work though." ) );
}
```

Reply:
If you fail at 0 you get a `m_bad` message so you know you failed, if you fail more thant that you don't even know you failed. Except with some meta gaming since he message is not the same as success.
`add_msg( m_info, _( "You put the CBM in the pouch and close it." ) );` Is a failure message, IRL if you close the pouch before putting it in the autoclave the content won't be properly sterilized.

Reply:
The current logic is this: print `You put the CBM in the pouch and close it.` only if success is less than 0. If success is strictly equals to 0, then print only message about quality of the work, but in this case there isn't any indication of what work in question was done.
I want to suggest printing `You put the CBM in the pouch and close it.` in case of success less than or equals to 0 to indicate what you've done, and print any other flavor text in specific cases like success strictly equals to zero.

Reply:
I see, yeah that makes sense